package.json

```diff
@@ -1,14 +1,14 @@
 {
     "name": "erlang_ls",
```

Comment:
Should the name of the extension be changed as well?

Reply:
Let' leave it for now. That will be the name of the extension, so it should be clear enough.